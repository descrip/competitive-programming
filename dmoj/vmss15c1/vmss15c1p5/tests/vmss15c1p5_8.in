10 993 988
apple 416 10 29
dynamicprogramming 218 9 14
ermergerd 928 3 7
internet 228 22 32
rhode 145 27 8
rotten 259 4 32
stang 464 6 7
sucks 337 14 23
thisisaname 682 17 11
yokoono 299 18 2
