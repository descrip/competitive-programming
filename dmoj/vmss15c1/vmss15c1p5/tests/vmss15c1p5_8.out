132064
apple 0
dynamicprogramming 0
ermergerd 132
internet 0
rhode 0
rotten 0
stang 0
sucks 0
thisisaname 0
yokoono 32
